x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/persistence/DataFrameAnalyticsConfigProvider.java

```diff
@@ -108,9 +111,11 @@ public void get(String id, ActionListener<DataFrameAnalyticsConfig> listener) {
     /**
      * @param ids a comma separated list of single IDs and/or wildcards
      */
-    public void getMultiple(String ids, ActionListener<List<DataFrameAnalyticsConfig>> listener) {
+    public void getMultiple(String ids, boolean allowNoMatch, ActionListener<List<DataFrameAnalyticsConfig>> listener) {
         GetDataFrameAnalyticsAction.Request request = new GetDataFrameAnalyticsAction.Request();
+        request.setPageParams(new PageParams(0, MAX_CONFIGS_SIZE));
```

Comment:
@droberts195 This here is used for memory tracking and before that it'd would be limited to 100 jobs. Unfortunately we're still limited but now the limit is 10K.

Reply:
Good spot - thanks for fixing it.